src/op_analytics/datapipeline/models/compute/udfs.py

```diff
@@ -15,6 +15,9 @@ def create_duckdb_macros(duckdb_client: duckdb.DuckDBPyConnection):
 
     CREATE OR REPLACE MACRO wei_to_gwei(a)
     AS a::DECIMAL(28, 0) * 0.000000001::DECIMAL(10, 10);
+    
+    CREATE OR REPLACE MACRO gwei_to_eth(a)
```

Comment:
modified decimal precision here, it wasn't allowing for a to be a decimal before